src/js/components/NavButton.jsx

```diff
@@ -50,15 +44,19 @@ const NavButton = ({
   let displayName = suffix === 'syntacticalvalidity' ? '' : suffix
   displayName = suffix !== 'submission' ? `${displayName} Edits` : displayName
 
-  return (
+  return [
     <Link
+      key="0"
       className={`NavButton usa-button ${className || ''}`}
       tabIndex={className === 'hidden' ? -1 : 0}
       to={`${base}/${suffix}`}
     >
       {`Review ${displayName}`}
-    </Link>
-  )
+    </Link>,
+    <div key="1" className="NavSpinner">
```

Comment:
Can we wrap this whole `<div>` in the `spinOn` check? It would just render an empty `<div>` anyway.